spec/koans.js

```diff
@@ -42,39 +43,39 @@ describe('`const` is like `let` plus read-only. ', () => {
 
   describe('scalar values are read-only', () => {
     it('number are read-only', () => {
-      // const constNum = 0;
-      // constNum = 1;
+      const constNum = 0;
+      let constNum = 1;
 
-      //expect(constNum).toBe(0);
+      expect(constNum).toBe(0);
     });
 
     it('string are read-only', () => {
-      // const constString = "I am a const";
-      // constString = "Cant change you?";
+      const constString = "I am a const";
+      let constString = "Cant change you?";
 
-      //expect(constString).toBe("I am a const");
+      expect(constString).toBe("I am a const");
     });
 
   });
 
-  /*var, let or const? notChangeable = 23;*/
+  const notChangeable = 23;
 
   it('const scope leaks too', () => {
-    //expect(notChangeable).toBe(23);
+    expect(notChangeable).toBe(23);
   });
 
   describe('complex types are NOT fully read-only', () => {
 
     it('arrays is not fully read-only', () => {
       const arr = [42, 23];
 
-      //expect(arr[0]).toBe(0);
+      expect(arr[0]).toBe(0);
```

Comment:
The point of this test is to show that you cannot redeclare a value for a variable declares with const, however for arrays you can modify some elements inside the array. (Same with objects, you can modify some properties inside the object)
in this case, you can do 
```js
    it('arrays is not fully read-only', () => {
      const arr = [42, 23];
      arr[0] = 0;
      expect(arr[0]).toBe(0);
    });
```